### ISSUE TITLE
[Bug]: t2i style adapter only work with "DDIM", "PLMS", "UniPC" sampler

### ISSUE THREAD

**Issue by jmaxzh** [Mikubill/sd-webui-controlnet#909]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

the t2i style adapter only work with "DDIM", "PLMS", "UniPC" sampler,  other t2i adapter can work normally with any sampler.

There is no error message.

Preprocessor: t2ia_style_clipvision
Model: t2iadapter_style_sd14v1
Used model for generation: anything-v4.0-pruned-fp16.safetensors

### Steps to reproduce the problem

Its in the description

### What should have happened?

t2i style adapter can work normally with any sampler.

### Commit where the problem happens

webui: latest
controlnet: latest

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--update-check --xformers --listen --port 8080 --medvram
```

### Console logs

```Shell
Steps: 20, Sampler: DPM2 Karras, CFG scale: 7, Seed: 2325472963, Size: 512x512, Model hash: d5f8393fea, Model: anything-v4.0-pruned-fp16, ControlNet-0 Enabled: True, ControlNet-0 Module: t2ia_style_clipvision, ControlNet-0 Model: t2iadapter_style_sd14v1 [202e85cc], ControlNet-0 Weight: 1, ControlNet-0 Guidance Start: 0, ControlNet-0 Guidance End: 1
```

**Comment by lllyasviel** [comment]
can anyone reproduce?

**Comment by tkalayci71** [comment]
not sure if --always-batch-cond-uncond  needed for --medvram

**Comment by lengsehuoyan** [comment]
I had the same problem

**Comment by lllyasviel** [comment]
does it has any error logs?

**Comment by lllyasviel** [comment]
fixed in 1.1.105
